src/petals/bloom/modeling_utils.py

```diff
@@ -0,0 +1,74 @@
+"""
```

Comment:
This file is not new. It was renamed from model.py, but git does not recognize the diff